kinto/views/__init__.py

```diff
@@ -29,4 +30,10 @@ def object_exists_or_404(request, collection_id, object_id, parent_id=''):
                            object_id=object_id)
     except exceptions.RecordNotFoundError:
         # XXX: We gave up putting details about parent id here (See #53).
-        raise httpexceptions.HTTPNotFound()
+        detail_dict = {
+                "id": collection_id,
+                "resource_name": "collection"
+            }
```

Comment:
@glasserc  Even i couldn't find anything for buckets.
@leplatrem id is to object_id I understood from #710 but what exactly does a resource_name define?